bigtop-packages/src/rpm/kafka/SPECS/kafka.spec

```diff
@@ -66,6 +66,7 @@ Source2: install_%{kafka_name}.sh
 Source3: kafka-server.svc
 Source4: init.d.tmpl
 Source6: kafka.default
+#BIGTOP_PATCH_FILES
```

Comment:
Just to be sure, are those comments needed or added while testing?

Reply:
We need the comments for rpm packaging to apply `patch*.diff` files. Bigtop replaces the `#BIGTOP_PATCH_FILES` in spec file with list of patches (and replaces `#BIGTOP_PATCH_COMMANDS` with command lines applying them. https://github.com/apache/bigtop/blob/master/packages.gradle#L590-L607

While we have similar trick for deb packaging, we usually don't need modify sources of each package.
https://github.com/apache/bigtop/blob/master/packages.gradle#L438-L447